### PR TITLE
Update python3-adafruit-blinka_6.2.2.bb

### PR DESCRIPTION
**Pull request by iaiagianluca**
disabled qa, to allow shipping 32bit binaries also in a 64bit rpi machines

### CODE REVIEW

File: dynamic-layers/openembedded-layer/recipes-devtools/python/python3-adafruit-blinka_6.2.2.bb [hunk 1]
```diff
@@ -11,14 +11,8 @@ S = "${WORKDIR}/git"
...
-
-do_install:append() {
-# it ships ./bcm283x/pulseio/libgpiod_pulsein which is a prebuilt
-# 32bit binary therefore we should make this specific to 32bit rpi machines (based on bcm283x) only
-    if [ ${@bb.utils.contains('TUNE_FEATURES', 'callconvention-hard', '1', '0', d)} = "0" ]; then
-        rm -rf ${D}${PYTHON_SITEPACKAGES_DIR}/adafruit_blinka/microcontroller/bcm283x
-    fi
-}
+# to disable qa errors when shipping 32bit bynaries in 64bit rpi machines
+do_package_qa[noexec] = "1"
```
Comment (agherzan) [inline dynamic-layers/openembedded-layer/recipes-devtools/python/python3-adafruit-blinka_6.2.2.bb:15]: This solves the problem, but does that by throwing a blanket on a QA task. I would look for a better solution here.

**Review by agherzan — CHANGES_REQUESTED** [review]
Also, please note https://meta-raspberrypi.readthedocs.io/en/latest/contributing.html on the expectations of git log format (for example). Thanks!

**Review by iaiagianluca — COMMENTED** [review]
changed files permissions to 0644 instead removing it

File: dynamic-layers/openembedded-layer/recipes-devtools/python/python3-adafruit-blinka_6.2.2.bb (hunk 1 above)
Comment (kraj) [reply dynamic-layers/openembedded-layer/recipes-devtools/python/python3-adafruit-blinka_6.2.2.bb]: what will that get us ?

**Review by agherzan — CHANGES_REQUESTED** [review]
This MR needs some cleanup:
* squash things in one commit
* rebase to drop merge commits
* have a clear git log rationale of what and why
* adhere to https://meta-raspberrypi.readthedocs.io/en/latest/contributing.html (for example, line length recommendation in git log)

Once again, thanks for your contribution.

**Pull request closed without merging**